packages/p2p/src/components/buy-sell/buy-sell-table.jsx

```diff
@@ -5,32 +5,33 @@ import { localize }  from 'Components/i18next';
 import { BuyTable }  from './buy-table.jsx';
 import { SellTable } from './sell-table.jsx';
 
-export const BuySellTable = ({ table_type, setSelectedAd, exchange_amount, exchange_to_currency }) => {
+export const BuySellTable = ({ table_type, setSelectedAd }) => {
+
     const is_buy = table_type === 'buy';
+    const [offer_currency, setOfferCurrency] = React.useState([]);
```

Comment:
offer_currency here refers to a string? so it shouldn't be initialized as an array

```suggestion
    const [offer_currency, setOfferCurrency] = React.useState('');
```